code/modules/admin/sql_ban_system.dm

```diff
@@ -680,6 +702,8 @@
 					qdel(query_edit_ban_get_player)
 					return
 		qdel(query_edit_ban_get_player)
+	if(global_ban && (global_ban != old_globalban))
```

Comment:
I still am unsure as to what I should put here

Reply:
this is literally all that's left and what's causing the error, please help me get this done and this will be done for good.

Reply:
I don't know why you think anything at all needs to go here?

Reply:
I thought this was how the ban editing system worked but if it's not then I'll just delete this line and it'll be done

Reply:
You probably need to update this line ```var/datum/DBQuery/query_edit_ban = SSdbcore.NewQuery("UPDATE [format_table_name("ban")] SET expiration_time = IF('[duration]' LIKE '', NULL, bantime + INTERVAL [duration ? "[duration]" : "0"] [interval]), applies_to_admins = [applies_to_admins], reason = '[reason]', ckey = IF('[player_ckey]' LIKE '', NULL, '[player_ckey]'), ip = INET_ATON(IF('[player_ip]' LIKE '', NULL, '[player_ip]')), computerid = IF('[player_cid]' LIKE '', NULL, '[player_cid]'), edits = CONCAT(IFNULL(edits,''),'[sanitizeSQL(usr.client.key)] edited the following [jointext(changes_text, ", ")]<hr>') WHERE [where]")```